packages/perseus-editor/src/widgets/interaction-editor/interaction-editor.tsx

```diff
@@ -59,6 +46,21 @@ type DefaultProps = {
     graph: Props["graph"];
 };
 
+const defaultInteractionProps: DefaultProps = {
```

Comment:
Given this variable is only used once, I'd be in favour if you wanted to just inline it.